### ISSUE TITLE
Failed to install on 3.6.4 on Windows 10

### ISSUE THREAD

**Issue by BramVanroy** [pettarin/ipapy#1]
Tried installing this module through pip but got an installation error. Full trace:

```
PS C:\Users\bmvroy> pip install ipapy
Collecting ipapy
  Using cached https://files.pythonhosted.org/packages/ae/d2/1af2bf40023d0cbc5d5cb9477938b923b8dc2eae479e5f0d1b4350db72be/ipapy-0.0.8.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\bmvroy\AppData\Local\Temp\pip-install-2wtx0rh1\ipapy\setup.py", line 28, in <module>
        long_description=open("README.rst", "r").read(),
      File "c:\python\python36\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 6127: character maps to <undefined>

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\bmvroy\AppData\Local\Temp\pip-install-2wtx0rh1\ipapy\
```

As you can see, it's an encoding error. Probably this is related to a file opening somewhere where the encoding is not explictly said to UTF-8. By default, Python uses the OS's default/the user's preferred encoding: `locale.getpreferredencoding()`. I will take a look later, but I already share the bug now.

**Comment by BramVanroy** [comment]
Pull https://github.com/pettarin/ipapy/pull/2

**Comment by pettarin** [comment]
Merged #2, closing.
